aviary/subsystems/aerodynamics/flops_based/test/test_mux_component.py

```diff
@@ -133,7 +133,7 @@ def test_mux_multiengine(self):
         prob.set_val(Aircraft.Nacelle.LAMINAR_FLOW_LOWER, np.array([0.10, 0.04]))
 
         prob.run_model()
-        prob.check_partials(compact_print=True, method="cs")
+        # prob.check_partials(compact_print=True, method="cs")
```

Comment:
Is this purposefully commented out?

Reply:
I didn't understand why we do `check_partials` twice. Now, I keep it in.